proto/tendermint/store/types.proto

```diff
@@ -7,3 +7,8 @@ message BlockStoreState {
   int64 base   = 1;
   int64 height = 2;
 }
+
+message TxStatus {
+  int64 height    = 1;
+  int64 index     = 2;
+}
```

Comment:
[nit] this seems more like `TxLocation`  because it identifies where a transaction exists in a block but it doesn't contain any info about the status of the transaction. Should we rename the type?

Reply:
[suggestion] Could we also have a comment about what `index` represents?

Reply:
I also agree with Rootul's point that `TxStatus` contains information pertains to a transaction already included in one of the previous blocks. However, the term `Status` seems to imply the current state of a transaction that is still being processed.